tigerpath/majors_and_certificates/majors/CLA_2018.json

```
@@ -0,0 +1,373 @@
+              "course_list": [
+                "ART 200",
+                "ART 202",
+                "ART 203",
+                "ART 204",
+                "ART 300",
+                "ART 301",
+                "ART 302",
+                "ART 305",
+                "ART 306",
+                "ART 308",
+                "ART 401",
+                "ART 410",
+                "CLA *",
+                "CLG 2*",
+                "CLG 3*",
+                "LAT 2*",
+                "LAT 3*",
+                "HLS 346",
+                "HIS 290",
+                "HIS 343",
+                "HIS 355",
+                "HUM 205",
+                "HUM 216",
+                "HUM 217",
+                "NES 331",
+                "NES 205",
+                "NES 300",
+                "NES 301",
+                "NES 332",
+                "NES 335",
+                "PHI 205",
+                "PHI 300",
+                "PHI 301",
+                "PHI 335",
+                "POL 301",
+                "REL 230",
+                "REL 231",
+                "REL 251",
+                "REL 252",
+                "REL 337",
+                "REL 340"
+              ]
+            },
+            {
+              "name": "Ancient Greek or Latin",
+              "max_counted": 1,
+              "min_needed": 1,
+              "explanation": "Each student must successfully complete either Ancient Greek or Latin to the level of 108 or achieve an equivalent level of knowledge, as demonstrated through test scores (SAT, AP), a placement exam, or coursework (including through a summer language program). However, at least one language course must be taken at Princeton.",
+              "course_list": [
+                "LAT 108",
+                "CLG 108"
+              ]
+            },
+            {
+              "name": "Departmentals",
+              "max_counted": 1,
+              "min_needed": 2,
+              "explanation": "Eight departmental courses are required. These must include one survey course on ancient Greek history (CLA 216, CLA 217) and one survey course on Roman history (CLA 218, CLA 219); two seminars at the 300 level focusing on ancient history (e.g., CLA 326, CLA 327: Topics in Ancient History; CLA 324: Classical Historians and their Philosophies of History; CLA 325: Roman Law); one course on material culture (e.g., ART 202: Greek Art; ART308: Roman Cities and Countryside); two courses focusing in whole or in part on classical civilization (see list below); and one history course on a non-classical pre-nineteenth century civilization (e.g., EAS 335: Early Chinese History to 221; NES 220: Jews, Muslims, and Christians in the Middle Ages; HIS 350 History of France, 1685-1800). Students must also take the Junior Seminar in the fall of their junior year.",
+              "req_list": [
+                {
+                  "name": "Ancient Greek History",
+                  "max_counted": null,

```

Based on them specifying a specific distribution of 8 courses, it seems that they want exactly 1 course from this category and not more. 
```suggestion
                  "max_counted": 1,
```